RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -13,4 +13,14 @@ def customizeHitRecoveryInGluedDetOn(process):
        process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
    return process
 
+def customizeHitRecoveryInGluedDetTkSeedsOnly(process):
+   process = customizeMinPtForHitRecoveryInGluedDet(process,0.9)
+   if hasattr(process, "Chi2MeasurementEstimatorForP5"): # keep disabled for cosmics
+       process.Chi2MeasurementEstimatorForP5.MinPtForHitRecoveryInGluedDet = 100000
+   for X in "muonSeededMeasurementEstimatorForOutIn", "hitCollectorForOutInMuonSeeds":
```

Comment:
I think the only really relevant one is muonSeededMeasurementEstimatorForOutIn since InOut can't make new tracks and the seeding mainly looks at the outer TOB layers while HIPM affects only the double-sided layers (and I thought that the HIPM era was not changing the `hitCollectorForOutInMuonSeeds` while the custimise was).
That said, I'll check now and synchronize the two implementations.